composites/Plugin/Shared/components/LanguageNotice.js

```diff
@@ -0,0 +1,90 @@
+/* External dependencies */
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { __, sprintf } from "@wordpress/i18n";
+import interpolateComponents from "interpolate-components";
+
+/* Internal dependencies */
+import { makeOutboundLink } from "../../../../utils/makeOutboundLink";
+import colors from "../../../../style-guide/colors";
+
+const YoastLanguageNotice = styled.p`
+	margin: 1em 0;
+`;
+
+const ChangeLanguageLink = makeOutboundLink( styled.a`
+	color: ${ colors.$palette_blue_medium };
```

Comment:
We shouldn't use the colors prefixed with `palette_`, and only use their aliases. However, wondering it this link should just inherit the colors from WordPress as we do for other links (e.g. the "Go Premium" link)

Reply:
Same consideration for the links colors above, still needs an answer. /Cc @hedgefield